_data/hosting.yml

```
@@ -451,6 +451,13 @@ websites:
       - totp
     doc: https://support.stackpath.com/hc/en-us/articles/360022992631
 
+  - name: switchplus
```

```suggestion
  - name: Swizzonic (formerly switchplus)
```
They have since changed names.

---

Do we really want include the former name?

---

I included their old name for the edge case that people may search the site use that name.

---

The URL is included in the search terms, so using _Swizzonic_ as the name will be sufficient.